about.html

```diff
@@ -1,5 +1,26 @@
 --- 
 layout: default
 title: About a11y Berlin
-description: a11y Berlin is an usergroup about accessibility and inclusive design.
+description: a11y Berlin is a usergroup about accessibility and inclusive design.
```

Comment:
Minor content suggestion – could we please change `a11y` with first letter capitalized everywhere where we have it as a part of the meet-up name (including the `<title>`) 👉 `A11y Berlin`?

Reply:
Changed it.